CMakeLists.txt

```diff
@@ -572,47 +572,78 @@ if (USE_PYTHON)
+                  re.split(r'\\\\|/', x.replace(sys.prefix, '')) if p][0] for x in sys.path if x]))"
+              OUTPUT_VARIABLE PY_SUB_PATHS
+              OUTPUT_STRIP_TRAILING_WHITESPACE)
+            
+            # remove prefix from each python path
+            # and recursively copy not overwrite existing files
+            # python path locations
+            # we strip prefix designated by python variable sys-prefix
+            # on linux we get something like "/usr"
+            # on windows we get path to installation folder
+            foreach(PY_PATH ${PY_PATHS})
+                string(REPLACE "${PY_PREFIX}" "" PY_ALT_PATH "${PY_PATH}")
+                list(APPEND PYTHON_WRAPPER_PATHS ".${PY_ALT_PATH}")
+                # copy only if dir does not exists
+                if(NOT EXISTS "${TARGET_PREFIX}/${PY_ALT_PATH}")
+                    message(STATUS "cmake -E copy_directory ${PY_PREFIX}${PY_ALT_PATH} ${TARGET_PREFIX}${PY_ALT_PATH}")
+                    execute_process(
+                      COMMAND cmake -E copy_directory ${PY_PREFIX}/.${PY_ALT_PATH} ${TARGET_PREFIX}/${PY_ALT_PATH}
+                     )
+                 endif()
+            endforeach(PY_PATH)
             execute_process(
-                COMMAND python -c "import sys; print sys.exec_prefix"
-                OUTPUT_VARIABLE PY_EXEC_PREFIX
-                OUTPUT_STRIP_TRAILING_WHITESPACE)        
-            if (NOT (${PY_PREFIX} STREQUAL ${PY_EXEC_PREFIX}) )
-                message(WARNING "Python exec_prefix (${PY_EXEC_PREFIX}) different then prefix (${PY_PREFIX}). Not supported.")
-            endif()
+              COMMAND chmod -R 764 ${TARGET_PREFIX}/
+             )
+            
             message(STATUS "Python prefix: ${PY_PREFIX}")
             message(STATUS "Python target prefix: ${TARGET_PREFIX}")
+            message(STATUS "Python wrapper paths: ${PYTHON_WRAPPER_PATHS}")
             flow_define(PYTHON_PREFIX ${PY_PREFIX})
+            flow_define(PYTHON_PATHS  ${PYTHON_WRAPPER_PATHS})
             
-            # copy python library
-            set(PYTHON_COPY_LIB_TARGET  lib/${PYTHON_SUBDIR_MINOR})
-            execute_process(
-                COMMAND mkdir -p ${TARGET_PREFIX}/lib 
-                )             
-            # recursive copy, not overwrite existing files
-            # we copy all and exclude some paths and files during install    
+            # copy flow123d python scripts and modules
             execute_process(
-                COMMAND cp -rn ${PY_PREFIX}/lib/${PYTHON_SUBDIR_MINOR} ${TARGET_PREFIX}/lib 
+                COMMAND ${TARGET_PREFIX}/lib/flow123d 
                 )
-            # copy flow123d python scripts and modules
             execute_process(
                 COMMAND cp -rT ${CMAKE_SOURCE_DIR}/src/python ${TARGET_PREFIX}/lib/flow123d 
                 )
-
-            # copy python include            
+                
+            list(APPEND PYTHON_WRAPPER_PATHS "./lib/flow123d")
```

Comment:
This is meaning less since PYTHON_WRAPPER_PATHS is not used further on. Anyway I do not understand why to add flow123d directory at all. The these directories are added into PYTHON_PATH in PythonLoader and should not be used before.